kube-client/src/config/file_config.rs

```diff
@@ -262,6 +266,16 @@ impl Kubeconfig {
         Ok(merged_docs.unwrap_or_default())
     }
 
+    /// Read a Config from an arbitrary YAML document
```

Comment:
```suggestion
    /// Read a Config from an arbitrary YAML string
```

missed this part. input could be multi-doc.